activestorage/test/models/record_test.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require "test_helper"
+require "database/setup"
+
+class ActiveStorage::RecordTest < ActiveSupport::TestCase
+  setup do
+    @old_prefix = ActiveRecord::Base.table_name_prefix
+    @old_suffix = ActiveRecord::Base.table_name_suffix
+
+    ActiveRecord::Base.table_name_prefix = @prefix = "prefix_"
+    ActiveRecord::Base.table_name_suffix = @suffix = "_suffix"
+
+    @descendants = ActiveStorage::Record.descendants
+    @descendants.map(&:reset_table_name)
```

Comment:
calling `reset_table_name` is necessary for affixes to be taken into consideration